src/pages/home/home.ts

```diff
@@ -12,76 +12,72 @@ import { InventoryData } from '../../providers/inventory-data';
   templateUrl: 'home.html'
 })
 export class HomePage {
-  actions = Actions;
-  segment: Actions = this.actions.rent;
-  tag: string = '';
 
   constructor(
     public navCtrl: NavController,
     public alertCtrl: AlertController,
     public notifications: Notifications,
     public inventoryData: InventoryData,
-    public barcodeScanner: BarcodeScanner
+    public barcodeScanner: BarcodeScanner,
+    public platform: Platform
   ) { }
 
-  onNext() {
-    if (this.tag) {
-      this.inventoryData.getItem(this.tag).subscribe(
-        (item: any) => this.pushPage(item, item.available === 1),
-        err => this.notifications.showToast(err)
-      );
-    }
-  }
-
-  pushPage(item, available: boolean) {
-    if (this.segment === this.actions.rent && !available) {
-      let alert = this.alertCtrl.create({
-        title: Messages.itemAlreadyRented,
-        message: 'Do you want to return it instead?',
-        buttons: [
-          {
-            text: 'Cancel',
-            role: 'cancel'
-          },
-          {
-            text: 'Return Item',
-            handler: () => {
-              this.segment = this.actions.return;
-              this.pushPage(item, available);
-            }
-          }
-        ]
-      });
-
-      alert.present();
-    } else if (this.segment === this.actions.return && available) {
-      let alert = this.alertCtrl.create({
-        title: Messages.itemNotRented,
-        subTitle: 'Cannot return the item',
-        buttons: ['OK']
-      });
+  pushPage(tag: string) {
```

Comment:
Should you rename all the `tag` variables to `barcode` for consistency and clarity?